apps/users/handlers.py

```diff
@@ -52,7 +52,7 @@ async def create_user(
             request: Request,
             user: CreateUserIn,
             session: AsyncSession):     # TODO refactor this method
-        hashed_password = get_hashed_password(user.password)
+        hashed_password, faculties = get_hashed_password(user.password), []
```

Comment:
Not sure that I understood the `faculties` variable here

Reply:
Tuple equals tuple

Reply:
But in last version faculties had deleted